lib/src/base/accessor.dart

```diff
@@ -351,7 +409,170 @@ class Accessor extends GltfChildOfRootProperty {
+    if (totalOffset.remainder(componentLength) != 0) {
+      if (context != null) {
+        context.addIssue(LinkError.accessorTotalOffsetAlignment,
+            args: [totalOffset, componentLength]);
+      } else {
+        return false;
+      }
+    }
+
+    // Length
+    if (byteOffset > bufferView.byteLength) {
+      if (context != null) {
+        context.addIssue(LinkError.accessorTooLong, name: BYTE_OFFSET, args: [
+          byteOffset,
+          byteLength,
+          _bufferViewIndex,
+          bufferView.byteLength
+        ]);
+      } else {
+        return false;
+      }
+    } else if (byteOffset + byteLength > bufferView.byteLength) {
+      if (context != null) {
+        context.addIssue(LinkError.accessorTooLong, args: [
+          byteOffset,
+          byteLength,
+          _bufferViewIndex,
+          bufferView.byteLength
+        ]);
+      } else {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  static List<int> _typedViewIndices(
+      int componentType, ByteBuffer buffer, int offsetInBytes, int length) {
+    assert(gl.COMPONENT_TYPES.contains(componentType));
+    if (buffer == null ||
+        buffer.lengthInBytes <
+            offsetInBytes + gl.componentTypeLength(componentType) * length) {
+      return null;
+    }
+    switch (componentType) {
+      case gl.UNSIGNED_BYTE:
+        return Uint8List.view(buffer, offsetInBytes, length);
+      case gl.UNSIGNED_SHORT:
+        return Uint16List.view(buffer, offsetInBytes, length);
+      case gl.UNSIGNED_INT:
+        return Uint32List.view(buffer, offsetInBytes, length);
+      default:
+        return null;
+    }
+  }
+
+  static Float32List _typedViewFloat(
+      int componentType, ByteBuffer buffer, int offsetInBytes, int length) {
+    assert(gl.COMPONENT_TYPES.contains(componentType));
+    if (buffer == null ||
+        buffer.lengthInBytes <
+            offsetInBytes + gl.componentTypeLength(componentType) * length) {
+      return null;
+    }
+    switch (componentType) {
+      case gl.FLOAT:
+        return Float32List.view(buffer, offsetInBytes, length);
+      default:
+        return null;
+    }
+  }
+
+  static List<int> _typedViewInt(
+      int componentType, ByteBuffer buffer, int offsetInBytes, int length) {
+    assert(gl.COMPONENT_TYPES.contains(componentType));
+    if (buffer == null ||
+        buffer.lengthInBytes <
+            offsetInBytes + gl.componentTypeLength(componentType) * length) {
+      return null;
+    }
+    switch (componentType) {
+      case gl.BYTE:
+        return Int8List.view(buffer, offsetInBytes, length);
+      case gl.UNSIGNED_BYTE:
+        return Uint8List.view(buffer, offsetInBytes, length);
+      case gl.SHORT:
+        return Int16List.view(buffer, offsetInBytes, length);
+      case gl.UNSIGNED_SHORT:
+        return Uint16List.view(buffer, offsetInBytes, length);
+/*    case gl.INT:
+      return Int32List.view(buffer, offsetInBytes, length);*/
```

Comment:
Is this supposed to be commented out?  I mean I realize this is just moved code from elsewhere, but it looks odd.

Reply:
This was commented out to increase coverage, since glTF accessors don't support `sint32`. Could as well be removed.